thrall/test/syndication/SyndicationRightsOpsTestsBase.scala

```diff
@@ -18,6 +18,10 @@ import scala.concurrent.duration._
 
 trait SyndicationRightsOpsTestsBase extends FreeSpec with Matchers with Fixtures with BeforeAndAfterAll with ScalaFutures with DockerKit with DockerTestKit with DockerKitSpotify {
 
+  val testProps = getTestProperties
+  val es6TestUrl = testProps.getOrElse("es6.test.url", "http://localhost:9206")
```

Comment:
As of https://github.com/guardian/grid/pull/2565 we've removed the ES1 cluster completely, that is we can use the more standard `9200` port here.

Reply:
Sorry must have missed that when rebasing